src/main/java/test/BaseTest.java

```diff
@@ -0,0 +1,45 @@
+package test;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+
+    public WebDriver driver;
+    public String baseUrl;
+    public boolean acceptNextAlert = true;
+    public StringBuilder verificationErrors = new StringBuilder();
+    public String browser;
+    public WebDriverWait webDriverWait;
+
+    @BeforeClass
+    public void setUP() {
+        browser = System.getProperty("browser");
+
+        System.setProperty("webdriver.chrome.driver",
+                "src" + File.separator + "main" + File.separator + "resources" + File.separator + "chromedriver.exe");
```

Comment:
comit resources to contain chromedriver and to be able to run on jenkins 
chromedriver_v86.exe